### PR TITLE
Disable javascript features

### PR DESCRIPTION
**Pull request by Antonio-Laguna**
This is a stab at #110

### CODE REVIEW

File: plugin-packs/postcss-preset-env/README.md [hunk 1]
```diff
@@ -87,6 +87,20 @@ features and supports **all** browsers.
...
+in which this isn't possible to polyfill a new behaviour with _just_ CSS.
+
+This is the current list of features that need a client library with a link
+to the polyfill's library.
+
+* `blank-pseudo-class`: [Plugin](https://github.com/csstools/postcss-plugins/blob/main/plugins/css-blank-pseudo) / [Polyfill](https://github.com/csstools/postcss-plugins/blob/main/plugins/css-blank-pseudo/README-BROWSER.md)
+* `has-pseudo-class`: [Plugin](https://github.com/csstools/postcss-plugins/blob/main/plugins/css-has-pseudo) / [Polyfill](https://github.com/csstools/postcss-plugins/blob/main/plugins/css-has-pseudo/README-BROWSER.md)
+* `prefers-color-scheme-query`: [Plugin](https://github.com/csstools/postcss-plugins/blob/main/plugins/css-prefers-color-scheme) / [Polyfill](https://github.com/csstools/postcss-plugins/blob/main/plugins/css-prefers-color-scheme/README-BROWSER.md)
+* `focus-within-pseudo-class`: [Plugin](https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-focus-within) / [Library](https://github.com/jsxtools/focus-within) / [Polyfill](https://github.com/jsxtools/focus-within/blob/master/README-BROWSER.md)
```
Comment (romainmenke) [inline plugin-packs/postcss-preset-env/README.md:100]: `focus-visible` also requires `js`

File: plugin-packs/postcss-preset-env/README.md [hunk 2]
```diff
@@ -358,6 +372,12 @@ postcssPresetEnv({
 
 The `debug` option enables debugging messages to stdout which should be useful to help you debug which features have been enabled/disabled and why.
 
+### disableClientSidePolyfills
+
+The `disableClientSidePolyfills` disables any feature that would need an extra browser library to be loaded into the page for it to work. 
+
+Note that you can still manually enable one of these features if you set them on the [`features` option](#features).
```
Comment (romainmenke) [inline plugin-packs/postcss-preset-env/README.md:379]:
```suggestion
Note that manually enabling/disabling features via the "feature" option overrides this flag.
```

**Review by romainmenke — COMMENTED** [review]
Small notes.

Looks great!

File: plugin-packs/postcss-preset-env/README.md (hunk 1 above)
Comment (Antonio-Laguna) [reply plugin-packs/postcss-preset-env/README.md]: Thanks for catching it!

**Review by romainmenke — APPROVED** [review]
LGTM!

**Pull request merged**